edge/pkg/edged/edged.go

```diff
@@ -254,6 +254,14 @@ type Config struct {
 }
 
 func init() {
+	_, err := os.Stat(DefaultRootDir)
+	if !os.IsExist(err) {
```

Comment:
`if err!=nil &&!os.IsExist(err)` here :)

Reply:
@fisherxu thank you,  we can also use `os.IsNotExist()` .   :)

Reply:
sure:)